### PR TITLE
added merge sort in php

### PR DESCRIPTION
**Pull request by Manasi219**
Added merge sort (Sorting Algorithm)
Implemented in PHP

### CODE REVIEW

**Comment by Manasi219** [comment]
Accept this Pull Request.. Merge Sort  program implemented using PHP is contributed..

File: Php/merge_sort.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+function merge_sort($my_array){
+	if(count($my_array) == 1 ) return $my_array;
+	$mid = count($my_array) / 2;
+    $left = array_slice($my_array, 0, $mid);
+    $right = array_slice($my_array, $mid);
+	$left = merge_sort($left);
```
Comment (diptangsu) [inline Php/merge_sort.php:7]: fix indentation

**Comment by diptangsu** [comment]
convert to 4 space indents

**Comment by Manasi219** [comment]
Fixed Indentation In merge_sort.php file.
Accept this full request.
Contributed: Merge Sort Program is implemented using PHP

**Pull request merged**